include/pagmo/algorithms/gaco.hpp

```diff
@@ -115,7 +115,7 @@ class PAGMO_DLL_PUBLIC gaco
      *
      * @throws std::invalid_argument if \p acc is not \f$ >=0 \f$, \p impstop is not a
      * positive integer, \p evalstop is not a positive integer, \p focus is not \f$ >=0 \f$,
-     * \p ker is not a positive integer, \p oracle is not positive, \p
+     * \p ker is not a positive integer \in [2, pop_size], \p oracle is not positive, \p
```

Comment:
@Sceki upon construction we don't know the population size, so it looks like that the requirement ``ker <= pop_size`` should be documented for the ``evolve()`` method instead?

Reply:
@bluescarni I do check it at the evolve and throw there, but I guess you want me to modify this doc and write only >=2 here?

Reply:
Yep it was only about the docstring. Basically, just leave ``throws if ker < 2`` in the ctor docs and add ``throws if ker is > pop_size`` in the evolve docs.

Reply:
@bluescarni Ok I pushed, have a look please :)

Reply:
Great, thanks a lot!